models/maat/R/model2netcdf.MAAT.R

```diff
@@ -141,7 +153,7 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
       out.year <- as.numeric(rep(year, sub.maat.output.dims[1]))
       output <- var_update(out.year, output, "Year", "Year", oldunits='YYYY', newunits=NULL, missval=-999, 
                            longname="Simulation Year", ncdims=ncdims)
-      output <- var_update(sub.maat.doy + day.steps, output, "FracJulianDay", "FracJulianDay", oldunits='Frac DOY', newunits=NULL, missval=-999, 
+      output <- var_update(tvals, output, "FracJulianDay", "FracJulianDay", oldunits = "Frac DOY", newunits = NULL, missval = -999, 
```

Comment:
Since you're defining `var_update` inside this function anyway (which I think is bad style, but no need to change), you might as well give it `missval` and `ncdims` as defaults to save you all the typing.